CHANGELOG

```diff
@@ -132,6 +132,9 @@ Integrations, testing and documentation
 - Switched from Artistic Style (astyle) version 2.05.1 to version 3.1 for code
   formatting. See the "make style" target.
 
+- New benchmark for the different assembly levels inspired by the CEED
```

Comment:
I am not sure to see the relation between this new assembly benchmark and the CEED Back-Off Problems.
Is this benchmark only focusing on assembly?

Reply:
No, it is benchmarking all the different parts: assembly, BP, BK; for the different levels of assembly: PA, EA, FA. So, this is mostly a more generic version of the ceed benchmark. Also instead of benchmarking only one point, it benchmarks a range of degrees of freedom. Do you want me to add this to the documentation?

Reply:
I improved the documentation, tell me if this is clear enough for you.